src/usr/local/www/interfaces.php

```diff
@@ -794,6 +797,9 @@ function remove_bad_chars($string) {
 	if (($_POST['spoofmac'] && !is_macaddr($_POST['spoofmac']))) {
 		$input_errors[] = gettext("A valid MAC address must be specified.");
 	}
+	if (($_POST['dhcp6-duid'] && empty($config['system']['global-v6duid']))) {
+		$input_errors[] = gettext("No System DUID has been entered. Please enter a valid DUID in System->Advanced->Networking");
+	}
```

Comment:
It should be a warning. It can be a static warning together with field description and can also include a link to system -> advanced -> network to make it easy for users